client/src/templates/Challenges/redux/create-question-epic.js

```diff
@@ -21,10 +21,10 @@ function filesToMarkdown(challengeFiles = {}) {
       return fileString;
     }
     const fileName = moreThanOneFile
-      ? `\\ file: ${challengeFile.contents}`
+      ? `\\ file: ${challengeFile.name}.${challengeFile.ext}\n`
```

Comment:
```suggestion
      ? `\* file: ${challengeFile.name}.${challengeFile.ext} */\n`
```

Reply:
Yea this is a lot better. If anyone copies the code then they won't face any errors.